client/capstone-app/src/component/DocList.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable jsx-a11y/heading-has-content */
+import React, { useState } from "react";
+import "./Doc.css";
+import { Link } from "react-router-dom";
+function Docdata() {
+  let data = [
```

Comment:
You should ideally keep the data in an separate file, now you are copying the data to all the components which is not good.